rllib/models/catalog.py

```diff
@@ -302,6 +304,11 @@ def get_model_v2(obs_space: gym.Space,
                 model_config["custom_model_config"] = \
                     model_config.pop("custom_options")
 
+            # Allow model kwargs to be overriden / augmented by
+            # custom_model_config.
+            customized_model_kwargs = dict(
```

Comment:
@sven1977 I believe that we should have been adding the args in this method, instead of in the torch template. Otherwise, policies that call get_model_v2() directly (e.g., DQN) miss passing the config properly as kwargs.